src/AccessibilityInsights.SharedUx/ViewModels/HierarchyNodeViewModel.cs

```diff
@@ -301,10 +302,8 @@ public HierarchyNodeViewModel(A11yElement ce, bool showUncertain, bool isLiveMod
         /// <param name="isLiveMode">Is node in live mode</param>
         private void UpdateIconInfoAndAutomationName(bool isLiveMode)
         {
-            StringBuilder sb = new StringBuilder();
-            sb.Append(this.Element.Glimpse);
-
             this.IconVisibility = Visibility.Collapsed;
+            string automationNameFormat = Resources.HierarchyNodeViewModel_AutomationNameDefaultFormat;
```

Comment:
The first attempt here set this to null, which then caused an error in live mode. The revised default is just {0}